.github/workflows/generate-bp.yml

```diff
@@ -3,6 +3,8 @@ name: Generate Baseline Profiles
 on:
   push:
     branches: [ main ]
+  pull_request:
```

Comment:
Remove this part again. We ONLY want to build baseline profiles for pushes on main as they take substantial time and this project is running on GitHub free tier.

Reply:
Thank you! I added a comment to explain the reason.